pkgs/development/libraries/libsmartcols/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, python3 }:
+
+stdenv.mkDerivation rec {
+  name = "libsmartcols";
+  version = "v2.36.1";
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig python3 ];
+
+  src = fetchFromGitHub {
+    owner = "karelzak";
+    repo = "util-linux";
+    rev = version;
+    sha256 = "0z7nv054pqhlihqiw0vk3h40j0cxk1yxf8zzh0ddmvk6834cnyxs";
+  };
+
+  configureFlags = ["--disable-all-programs" "--enable-libsmartcols"];
+
+  buildPhase = ''
+    make libsmartcols.la
+  '';
+
+  installTargets = [ "install-am" "install-pkgconfigDATA"];
+
+  meta = {
+    description = "libsmartcols - pretty output for everyone";
```

Comment:
descriptions shouldn't start with the package name - and something like the debian description might be a bit more meaningful:

> smart column output alignment library

Reply:
ah, I took it from  the authors blog post introducing it. But That is an easy change :)